README.md

```diff
@@ -1,15 +1,132 @@
 # vui-input
+[![Bower version][bower-image]][bower-url]
 [![NPM version][npm-image]][npm-url]
 [![Build status][ci-image]][ci-url]
 [![Dependency Status][dependencies-image]][dependencies-url]
 
 This component contains Sass mixins and CSS that can be used to style inputs.
 
+## Installation
+
+Install from NPM:
+```shell
+npm install vui-input
+```
+
+Install from Bower:
+```shell
+bower install vui-input
+```
+
+## Usage
+
+**Text:**
+
+```scss
+@import 'bower_components/vui-input/input.scss'; // or...
+
+@import "node_modules/vui-input/input.scss";
+
+.app {
+	input {
```

Comment:
I think we want to target this mixin a bit more so it doesn't get applied to checkboxes and radios.